### DISCUSSION TITLE
Add Compatibility with Ninja Tables

### DISCUSSION THREAD

**Discussion by rizaardiyanto1412** [publishpress/PublishPress-Permissions#872] [Feature Requests]
Currently we cannot limit to specific tables in Ninja Tables, I think it's not compatible yet. It will be great if we have compatibility with Ninja Table

**Comment by mitch-williams-src** [comment]
Setting specific permissions for Ninja Tables would be a big step towards making the PublishPress plugins fill more of my current needs.
